### ISSUE TITLE
A better Korean translation engine

### ISSUE THREAD

**Issue by isayj** [SubtitleEdit/subtitleedit#7005]
Hi there! 
Right now the translations are done by Google Translate V1 API. However, it does not perform well in a completely different language that is not alphabet-based. It would be a good idea to introduce the local engine for better performance. I would really want to get to know some subtitles of the original Korean texts(which thanks to the Whisper Speech Recognition embedded in SubtitleEdit) and I bet there might be many people out there who love KPOP also want this.
[Papago](https://papago.naver.com/) is an awesome translator created by Naver in Korea, which is a commonly-used search engine for the locals and thus supports powerful tools for any language translated into or from Korean. One can check more detail about the API [here](https://papago.naver.com/business).

**Comment by niksedk** [comment]
Good idea :)
I do not have enough time ATM.
I the mean time you could try DeepL and see if it's better than Google.
Also, you can try the copy-paste translate, which after a few tries will be pretty easy.

**Comment by rsmith02ct** [comment]
DeepL works quite well for Japanese (much better than Google Translate). Copy paste only works if you feed it unbroken sentences, not single lines.

**Comment by niksedk** [comment]
Hm, it's a bit hard to sign up to Naver when you live in Denmark...

**Comment by isayj** [comment]
> Hm, it's a bit hard to sign up to Naver when you live in Denmark...

Thanks for your time and effort of looking into it!
I'm sorry to hear that. As I have tried (I'm a foreigner as well), both the Facebook login option and the phone verification do not work out for me, which, I suppose, is the same case to you.
Unfortunately, I cannot find a proper way to contact their customer service other than a FAQ in dealing with the login issue. 

However, I used the translation very frequently without registering at all if you just aim at trying it out. Also, the [API request link](https://www.ncloud.com/support/question/service) should be open to any developers afaik.
